ONBOARDING.md

```diff
@@ -0,0 +1,138 @@
+## Getting Started
+
+### Core SDK
+
+First access the mParticle Core SDK repository via the following
+link: [mParticle GitHub Repo](https://github.com/mParticle/mparticle-android-sdk)
+You can easily "fork" the repository by clicking the **Fork** tab in the upper right corner, once
+that is done you should be able to see it in your own Github account under the **Repositories** tab.
+Open your forked version and in the green **Code** tab copy the https link. Go ahead and open
+Android Studio and select Project from Version Control or Get from VCS, paste the Repository link
+you just copied and select **Clone**.
+
+### Testing the Core SDK
+
+Once the project is loaded, run the default tests to validate that your environment is set up
+correctly and to check that the code is working correctly before making changes of your own.
+
+You can do so locally using the the following gradle wrapper commands in your terminal:
+
+For Lint Checks:
+`./gradlew lint`
+
+For Unit Tests:
+`./gradlew test`
+
+For Instrumented Tests:
+`./gradlew :android-core:cAT :android-kit-base:cAT --stacktrace`
+
+In some machines running windows you may need to type the commands without **./**
+eg. `gradlew lint`
+
+#### Issues with JAVA_HOME
+
+When running the tests for the first time, It is possible that you run into the following issue:
+
+**ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.**
+
+**Please set the JAVA_HOME variable in your environment to match the location of your Java
+installation.**
+
+Any version between Java 8 and 11 is required to execute gradle, **Any Java versions above that are
```

Comment:
Correct me if I'm wrong, but I believe Java 11 is required (won't work with Java 8/9 etc).

This isn't a requirement by us, but since we use AGP 7+, which requires Gradle 7+, which requires Android Studio Arctic Fox+, which requires Java 11, it's a defacto requirement